### PR TITLE
kcat: add page

### PR DESCRIPTION
**Pull request by ImShakthi**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/kcat.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`kcat -C -t {{topic}} -b {{brokers}}`
+
+- Consume messages starting with the oldest offset and exit after the last message is received:
+
+`kcat -C -t {{topic}} -b {{brokers}} -o beginning -e`
+
+- Consume messages as a Kafka consumer group:
+
+`kcat -G {{group_id}} {{topic}} -b {{brokers}}`
```
Comment (blueskyson) [inline pages/common/kcat.md:16]: Does it miss `-t` here?
```suggestion
`kcat -G {{group_id}} -t {{topic}} -b {{brokers}}`
```

**Comment by marchersimon** [comment]
@ImShakthi please update your Fork to use `main` as default branch, instead of `master`. We switched about half a year ago.

File: pages/common/kcat.md (hunk 1 above)
Comment (ImShakthi) [reply pages/common/kcat.md:16]: @blueskyson you are right, will add the missing `-t` and update the PR

**Comment by marchersimon** [comment]
But there's no need to hurry. You can still manually select `main` in every PR and it will work perfectly fine.

**Pull request closed without merging**